recipes/museotoolbox/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "museotoolbox" %}
+{% set version = "0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 30201a51d6033c1b191bd822741f7cbf073053926928d818ea874bfcad3f9917
+
+build:
+  noarch : python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - joblib
+    - matplotlib
+    - gdal >=2
+    - numpy >=1.7
+    - pip
+    - python
+    - scikit-learn
+    - scipy
```

Comment:
Do you need all of these at install time?

Reply:
Hello @scopatz ,
I updated the library needed for install time : https://github.com/conda-forge/staged-recipes/pull/10863/commits/fbc0aa4f54f489b6ec79a0f4ab66e39574c870f5
Kind regards,
Nicolas.